src/dispatch/database/manage.py

```diff
@@ -133,20 +134,20 @@ def init_database(engine):
         )
 
 
-def init_schema(*, engine, organization: Organization):
+def init_schema(*, engine: Engine, organization: Organization) -> Organization:
     """Initializes a new schema."""
     schema_name = f"{DISPATCH_ORGANIZATION_SCHEMA_PREFIX}_{organization.slug}"
 
-    if not engine.dialect.has_schema(engine, schema_name):
-        with engine.connect() as connection:
-            connection.execute(CreateSchema(schema_name))
+    with engine.begin() as connection:
+        connection.execute(CreateSchema(schema_name, if_not_exists=True))
 
     # set the schema for table creation
     tables = get_tenant_tables()
 
     schema_engine = engine.execution_options(
         schema_translate_map={
             None: schema_name,
+            "dispatch_core": "dispatch_core",
```

Comment:
Added to resolve [test failure](https://github.com/Netflix/dispatch/actions/runs/4624040233/jobs/8178465630?pr=3228)

This passes.

cc @kevgliss  @mvilanova

Reply:
This one seems wrong.